lib/faker/string.ex

```diff
@@ -24,4 +26,525 @@ defmodule Faker.String do
+    "1/0",
+    "0/0",
+    "-2147483648/-1",
+    "-9223372036854775808/-1",
+    "-0",
+    "-0.0",
+    "+0",
+    "+0.0",
+    "0.00",
+    "0..0",
+    ".",
+    "0.0.0",
+    "0,00",
+    "0,,0",
+    ",",
+    "0,0,0",
+    "0.0/0",
+    "1.0/0.0",
+    "0.0/0.0",
+    "1,0/0,0",
+    "0,0/0,0",
+    "--1",
+    "-",
+    "-.",
+    "-,",
+    "999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999",
+    "NaN",
+    "Infinity",
+    "-Infinity",
+    "INF",
+    "1#INF",
+    "-1#IND",
+    "1#QNAN",
+    "1#SNAN",
+    "1#IND",
+    "0x0",
+    "0xffffffff",
+    "0xffffffffffffffff",
+    "0xabad1dea",
+    "123456789012345678901234567890123456789",
+    "1,000.00",
+    "1 000.00",
+    "1'000.00",
+    "1,000,000.00",
+    "1 000 000.00",
+    "1'000'000.00",
+    "1.000,00",
+    "1 000,00",
+    "1'000,00",
+    "1.000.000,00",
+    "1 000 000,00",
+    "1'000'000,00",
+    "01000",
+    "08",
+    "09",
+    "2.2250738585072011e-308",
+    ",./;'[]\\-=",
+    "<>?:\"{}|_+",
+    "!@#$%^&*()`~",
+    "\u0001\u0002\u0003\u0004\u0005\u0006\u0007\b\u000e\u000f\u0010\u0011\u0012\u0013\u0014\u0015\u0016\u0017\u0018\u0019\u001a\u001b\u001c\u001d\u001e\u001f",
+    "",
+    "\t\u000b\f             ​    　",
+    "­؀؁؂؃؄؅؜۝܏᠎​‌‍‎‏‪‫‬‭‮⁠⁡⁢⁣⁤⁦⁧⁨⁩⁪⁫⁬⁭⁮⁯﻿￹￺￻𑂽𛲠𛲡𛲢𛲣𝅳𝅴𝅵𝅶𝅷𝅸𝅹𝅺󠀁󠀠󠀡󠀢󠀣󠀤󠀥󠀦󠀧󠀨󠀩󠀪󠀫󠀬󠀭󠀮󠀯󠀰󠀱󠀲󠀳󠀴󠀵󠀶󠀷󠀸󠀹󠀺󠀻󠀼󠀽󠀾󠀿󠁀󠁁󠁂󠁃󠁄󠁅󠁆󠁇󠁈󠁉󠁊󠁋󠁌󠁍󠁎󠁏󠁐󠁑󠁒󠁓󠁔󠁕󠁖󠁗󠁘󠁙󠁚󠁛󠁜󠁝󠁞󠁟󠁠󠁡󠁢󠁣󠁤󠁥󠁦󠁧󠁨󠁩󠁪󠁫󠁬󠁭󠁮󠁯󠁰󠁱󠁲󠁳󠁴󠁵󠁶󠁷󠁸󠁹󠁺󠁻󠁼󠁽󠁾󠁿",
+    "﻿",
+    "￾",
+    "Ω≈ç√∫˜µ≤≥÷",
+    "åß∂ƒ©˙∆˚¬…æ",
+    "œ∑´®†¥¨ˆøπ“‘",
+    "¡™£¢∞§¶•ªº–≠",
+    "¸˛Ç◊ı˜Â¯˘¿",
+    "ÅÍÎÏ˝ÓÔÒÚÆ☃",
+    "Œ„´‰ˇÁ¨ˆØ∏”’",
+    "`⁄€‹›ﬁﬂ‡°·‚—±",
+    "⅛⅜⅝⅞",
+    "ЁЂЃЄЅІЇЈЉЊЋЌЍЎЏАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯабвгдежзийклмнопрстуфхцчшщъыьэюя",
+    "٠١٢٣٤٥٦٧٨٩",
+    "⁰⁴⁵",
+    "₀₁₂",
+    "⁰⁴⁵₀₁₂",
+    "ด้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็ ด้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็ ด้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็็้้้้้้้้็็็็็้้้้้็็็็",
+    "'",
+    "\"",
+    "''",
+    "\"\"",
+    "'\"'",
+    "\"''''\"'\"",
+    "\"'\"'\"''''\"",
+    "<foo val=“bar” />",
+    "<foo val=“bar” />",
+    "<foo val=”bar“ />",
+    "<foo val=`bar' />",
+    "田中さんにあげて下さい",
+    "パーティーへ行かないか",
+    "和製漢語",
+    "部落格",
+    "사회과학원 어학연구소",
+    "찦차를 타고 온 펲시맨과 쑛다리 똠방각하",
+    "社會科學院語學研究所",
+    "울란바토르",
+    "𠜎𠜱𠝹𠱓𠱸𠲖𠳏",
+    "表ポあA鷗ŒéＢ逍Üßªąñ丂㐀𠀀",
+    "Ⱥ",
+    "Ⱦ",
+    "ヽ༼ຈل͜ຈ༽ﾉ ヽ༼ຈل͜ຈ༽ﾉ",
+    "(｡◕ ∀ ◕｡)",
+    "｀ｨ(´∀｀∩",
+    "__ﾛ(,_,*)",
+    "・(￣∀￣)・:*:",
+    "ﾟ･✿ヾ╲(｡◕‿◕｡)╱✿･ﾟ",
+    ",。・:*:・゜’( ☻ ω ☻ )。・:*:・゜’",
+    "(╯°□°）╯︵ ┻━┻)",
+    "(ﾉಥ益ಥ）ﾉ﻿ ┻━┻",
+    "┬─┬ノ( º _ ºノ)",
+    "( ͡° ͜ʖ ͡°)",
+    "😍",
+    "👩🏽",
+    "👾 🙇 💁 🙅 🙆 🙋 🙎 🙍",
+    "🐵 🙈 🙉 🙊",
+    "❤️ 💔 💌 💕 💞 💓 💗 💖 💘 💝 💟 💜 💛 💚 💙",
+    "✋🏿 💪🏿 👐🏿 🙌🏿 👏🏿 🙏🏿",
+    "🚾 🆒 🆓 🆕 🆖 🆗 🆙 🏧",
+    "0️⃣ 1️⃣ 2️⃣ 3️⃣ 4️⃣ 5️⃣ 6️⃣ 7️⃣ 8️⃣ 9️⃣ 🔟",
+    "🇺🇸🇷🇺🇸 🇦🇫🇦🇲🇸",
+    "🇺🇸🇷🇺🇸🇦🇫🇦🇲",
+    "🇺🇸🇷🇺🇸🇦",
```

Comment:
I wonder if we want to store emojis as multibytes. Thoughts?

Reply:
I'm on the fence about it. I like being able to see what characters the bytes actually are, but then again the entire point of a lot of these strings is that stuff messes up when trying to display them. Storing all the tricky stuff as multibytes would definitely get rid of any problems with displaying the source file. Do you have a preference either way?

Reply:
@igas @anthonator thoughts?

Reply:
I personally don't have preference.

Reply:
I'm going to assume we're good with the current implementation in order to move forward.